File: 프로그래머스/Level1/고득점 Kit/K번째수-정렬.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    
    for(int i=0; i<commands.size(); i++){
        int left = commands[i][0]-1;
        int right = commands[i][1]-1;
        int idx = commands[i][2]-1;
        
        vector<int> v;
        for(int j=left; j<=right; j++){
            v.push_back(array[j]);
        }
        
        sort(v.begin(), v.end());
        answer.push_back(v[idx]);
    }
    
    return answer;
}

File: c++/프로그래머스/Level1/고득점 Kit/K번째수-정렬.py
def solution(array, commands):
    answer = []
    
    for cmd in commands :
        i = cmd[0] - 1
        j = cmd[1] - 1
        k = cmd[2] - 1
        
        tmp_list = array[i:j+1]
        tmp_list = sorted(tmp_list)
        answer.append(tmp_list[k])
        
    return answer